contracts/chain-adapters/l2/WithdrawalHelperBase.sol

```diff
@@ -35,10 +44,28 @@ abstract contract WithdrawalAdapterBase is CircleCCTPAdapter, MultiCaller {
         ITokenMessenger _cctpTokenMessenger,
         uint32 _destinationCircleDomainId,
         address _l2TokenGateway,
-        address _tokenRecipient
+        address _tokenRecipient,
+        address _hubPool
```

Comment:
I think this needs a corresponding param update in the natspec above. Also on the naming, I think the SpokePool contracts typically use `crossDomainAdmin` to specify the owner/admin and `hubPool` to specify the recipient on mainnet for withdrawals, so this seems a bit inconsistent with that in the sense that `hubPool` has permission to execute admin functions here.

Reply:
Good catch. I added the natspec to reflect the new parameter. https://github.com/across-protocol/contracts/pull/605/commits/2056784034360ccdda33d025d7d86b16be170728. Also, I agree with renaming `hubPool`, so I just changed the functions to match the same ones as the spoke pool (i.e. use `crossDomainAdmin`).